core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -370,7 +423,13 @@ public void testInlineScriptInsertZeros() {
         }
     }
 
-    public void testIndexedScript() {
+    public void testStoredScript() {
+        assertAcked(client().admin().cluster().preparePutStoredScript()
+                .setId("my_script")
+                .setScriptLang(CustomScriptPlugin.NAME)
+                // Source is not interpreted but my_script is defined in CustomScriptPlugin
+                .setSource(new BytesArray("{ \"script\": \"_value0 + _value1 + _value2\" }")));
```

Comment:
I'd make the string here `"junk"` or `"not interpreted"` or something. If you don't read the file carefully it looks like the script is run because that is a valid looking script.